tests/base_d_asa/test_asset_update.py

```diff
@@ -36,9 +39,8 @@ def test_fail_unauthorized(
     base_d_asa_client_active: BaseDAsaClient,
 ) -> None:
     with pytest.raises(LogicError, match=err.UNAUTHORIZED):
-        base_d_asa_client_active.send.asset_update(
-            metadata=asset_metadata,
-            transaction_parameters=CommonAppCallParams(
-                sender=oscar.address, signer=oscar.signer
-            ),
+        base_d_asa_client_active.send.update.asset_update(
+            args=AssetUpdateArgs(metadata=asset_metadata),
+            params=CommonAppCallParams(sender=oscar.address, signer=oscar.signer),
+            compilation_params={},
```

Comment:
Why is this needed?

Reply:
Not needed anymore, patch in last pr (compilation params weren't properly accounting None case)